src/mono/mono/mini/interp/transform.c

```diff
@@ -11379,6 +11386,7 @@ mono_interp_transform_method (InterpMethod *imethod, ThreadContext *context, Mon
 		memcpy ((char*)imethod + start_offset, (char*)&tmp_imethod + start_offset, sizeof (InterpMethod) - start_offset);
 		mono_memory_barrier ();
 		imethod->transformed = TRUE;
+		imethod->method = method;
```

Comment:
Please merge `main` to pick up https://github.com/dotnet/runtime/pull/88738, and undo this change - it will break MonoJitInfo hashing for interpreter methods.

Reply:
merged